adagrams/game.py

```diff
@@ -1,11 +1,105 @@
+import random
+import copy
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    letter_pool_list = []
+    for key, value in LETTER_POOL.items():
+        letter = [key] * value
+        letter_pool_list.append(letter)
+    letter_pool_list = sum(letter_pool_list, [])
+
+
+    letter_bank = []
+    while len(letter_bank) < 10:
+        letter = random.choice(letter_pool_list)
+        letter_bank.append(letter)
+        letter_pool_list.remove(letter)
+
+    return(letter_bank)
+
+word_list = []
+def uses_available_letters(word, letters):
+    letters_copy = copy.deepcopy(letters)
+    word = word.upper()
+    for letter in word:
+        if letter in letters_copy:
+            letters_copy.remove(letter)
+        else:
+            return False
```

Comment:
Amazing work with this logic Angela! For each letter in the word you're checking if it is in the letter bank & removing it so you can check the next letter 🎉